lib/gi/configuration.rb

```diff
@@ -27,7 +27,6 @@ def connection
         # conn.request :curl, ::Logger.new(STDOUT), :warn
 
         # conn.response :logger, ::Logger.new(STDOUT), bodies: true
-        conn.response :betamocks if use_mocks?
```

Comment:
Do you plan to use mocks here as well? I'm unsure if the dev env has the correct configurations in dev to make the actual network request if this is not mocked

Reply:
Ok, it looks like the dev env has a some network settings through the fwdproxy for gids: https://github.com/department-of-veterans-affairs/vsp-platform-fwdproxy/blob/87ba39e6f9f1389088f1a508d0c9450b0368c43f/fwdproxy-config/fwdproxy-vagov-dev-vars.yml#L143-L150

Just want to check if mocking is necessary still

Reply:
Yes now that we're opening up vets-api to GIDS the mocking will no longer necessary. And it wasn't previously necessary for any other GI endpoints